tests/ssl-opt.sh

```diff
@@ -2931,7 +2922,7 @@ run_test    "Max fragment length: disabled, larger message" \
             -s "1 bytes read"
 
 requires_config_disabled MBEDTLS_SSL_MAX_FRAGMENT_LENGTH
-run_test    "Max fragment length DTLS: disabled, larger message" \
+run_test    "Max fragment length, DTLS: disabled, larger message" \
```

Comment:
I think this (and the ones above) doesn't require a constraint, because what the test checks is:
> "when the fragment length extension is enabled/disabled, for any `MAX_CONTENT_LEN`, check that setting +1 over the size limit is handled expectedly."

My understanding of the test above may not be entirely accurate, but the test passes for almost any `MAX_CONTENT_LEN` so I believe it is something close to it. Key here being "almost", as this and the ones above can fail if the message length is less than the certificate size (try setting IN/OUT to 512, the server log complained `certificate too large, 837 > 512`). Unfortunately I don't think I should hard-code this value, as it seems quite arbitrary. 

I thought this might be a user error ("please don't configure it so low"), and not something that we should handle (so it's fine if it fails the tests). I'd love some insight because I might be misunderstanding.

Reply:
Setting MAX_CONTENT_LEN to a small value will break our tests, but that's out of scope here. The minimum depends mostly on which test certificate the test case uses. This test case has no particular requirement on MAX_CONTENT_LEN, nothing more than the “Default” test case.